code/game/gamemodes/miniantags/space_turkey/space_turkey.dm

```diff
@@ -0,0 +1,117 @@
+							Objective #1: Hatch as many turkeys as possible.</b> (Lay your eggs in safe spots and protect them from the crew!)<br> \
+							<b>Objective #2: Make the crew as miserable as possible.</b> (No killing! The crew can't be miserable if they're dead!)"
+
+	var/egg_type = /obj/item/reagent_containers/food/snacks/egg/turkey
+
+/mob/living/simple_animal/hostile/space_turkey/proc/ghost_call()
+	notify_ghosts("A space turkey has been created at [get_area(src)]!", enter_link = "<a href=?src=[UID()];ghostjoin=1>(Click to enter)</a>", source = src, action = NOTIFY_ATTACK)
+
+/mob/living/simple_animal/hostile/space_turkey/New()
+	. = ..()
+
+	pass_flags &= ~PASSMOB
+	AddSpell(new /obj/effect/proc_holder/spell/targeted/lay_turkey_egg(null))
+	AddSpell(new /obj/effect/proc_holder/spell/targeted/gobble(null))
+	AddSpell(new /obj/effect/proc_holder/spell/targeted/turkey_tackle(null))
+
+/obj/item/reagent_containers/food/snacks/egg/turkey
+	name = "egg" //Originally had a snarky turkey egg description; removed to hide eggs with other eggs.
+	desc = "An egg!"
+
+#define TURKEY_HATCH_SPEED 30*7 //30 x # of minutes
+
+/obj/item/reagent_containers/food/snacks/egg/turkey/process()
+	if(isturf(loc))
+		amount_grown += 1
+		if(amount_grown >= TURKEY_HATCH_SPEED)
+			notify_ghosts("A space turkey is ready to hatch in [get_area(src)]!", enter_link = "<a href=?src=[UID()];ghostjoin=1>(Click to enter)</a>", source = src, action = NOTIFY_ATTACK)
+			STOP_PROCESSING(SSobj, src)
+	else
+		STOP_PROCESSING(SSobj, src)
+
+/obj/item/reagent_containers/food/snacks/egg/turkey/attack_ghost(mob/user)
+	if(src.amount_grown < TURKEY_HATCH_SPEED)
+		to_chat(user, "<span class='boldnotice'>This egg is not ready to hatch yet!</span>")
+		return
+	if(cannotPossess(user))
+		to_chat(user, "<span class='boldnotice'>Upon using the antagHUD you forfeited the ability to join the round.</span>")
+		return
+	if(jobban_isbanned(user, "Syndicate"))
+		to_chat(user, "<span class='warning'>You are banned from antagonists!</span>")
+		return
+	if(!src || QDELETED(src))
+		return
+	var/be_turkey = alert("Become a space turkey? (Warning, You can no longer be cloned!)",,"Yes","No")
+	if(be_turkey == "No" || !src || QDELETED(src))
+		return
+	visible_message("[src] hatches with a quiet cracking sound.")
+	var/mob/living/simple_animal/hostile/space_turkey/T = new(get_turf(src))
+	T.transfer_personality(user.client)
+	qdel(src)
+
+/mob/living/simple_animal/hostile/space_turkey/attack_ghost(mob/user)
+	if(cannotPossess(user))
+		to_chat(user, "<span class='boldnotice'>Upon using the antagHUD you forfeited the ability to join the round.</span>")
+		return
+	if(jobban_isbanned(user, "Syndicate"))
+		to_chat(user, "<span class='warning'>You are banned from antagonists!</span>")
+		return
+	if(!src || QDELETED(src))
+		return
+	var/be_turkey = alert("Become a space turkey? (Warning, You can no longer be cloned!)",,"Yes","No")
+	if(be_turkey == "No" || !src || QDELETED(src))
```

Comment:
what would happen if someone:
1. clicked the antaghud button
1. clicked the turkey
1. accepted the antaghud prompt, got the information they sought
1. accepted the turkey prompt, with the antag knowledge from antaghud

Reply:
I don't know. I copied that part from the Borer code. If it's an issue with the space turkeys, it'd probably be an issue with the borers as well.

Reply:
You should test this during the test merge, then, and fix it if that is indeed the case.

Reply:
Tested and fixed.

Note to self: fix the same bug in the borer code.